tests/appsec/api_security/test_schemas.py

```diff
@@ -147,8 +154,7 @@ def test_request_method(self):
         assert isinstance(schema, list)
         assert len(schema) == 1
         assert isinstance(schema[0], dict)
-        for key in ("content-length", "content-type", "x-option"):
-            assert key in schema[0]
```

Comment:
The weblog endpoint only documentation specifies that the parameters by use as the response headers https://github.com/DataDog/system-tests/blob/8da99ebdd9f1bd8269c61752f583873d75b4bb7f/docs/weblog/README.md#L139-L140

In this case, checking `"content-length", "content-type"` is not following the specification and rather accounting for the framework to include those 

I think in this case is best to just check for `x-option`